### PR TITLE
TW-491: refactor event name

### PR DESCRIPTION
**Pull request by NovakVlad2424**

### CODE REVIEW

File: src/components/bottom-sheet/bottom-sheet-action-button/bottom-sheet-action-button.tsx [hunk 1]
```diff
@@ -2,9 +2,10 @@ import { TouchableOpacity } from '@gorhom/bottom-sheet';
 import React, { FC } from 'react';
 import { Text, TouchableWithoutFeedbackProps } from 'react-native';
 
+import { TestIdProps } from '../../../interfaces/test-id.props';
 import { useBottomSheetActionButtonStyles } from './bottom-sheet-action-button.styles';
 
-interface Props extends Pick<TouchableWithoutFeedbackProps, 'style' | 'onPress'> {
+interface Props extends Pick<TouchableWithoutFeedbackProps, 'style' | 'onPress'>, TestIdProps {
```
Comment (unixvb) [inline src/components/bottom-sheet/bottom-sheet-action-button/bottom-sheet-action-button.tsx:8]: testId property not used

File: src/components/dropdown/dropdown.tsx [hunk 2]
```diff
@@ -28,7 +29,7 @@ export interface DropdownProps<T> extends Pick<FlatListProps<T>, 'keyExtractor'>
   onLongPress?: EmptyFn;
 }
 
-export interface DropdownValueProps<T> {
+export interface DropdownValueProps<T> extends TestIdProps {
```
Comment (unixvb) [inline src/components/dropdown/dropdown.tsx:32]: testId not used

File: src/components/icon/touchable-icon/touchable-icon.tsx [hunk 3]
```diff
@@ -4,12 +4,13 @@ import { GestureResponderEvent } from 'react-native';
 
 import { EmptyFn, EventFn } from '../../../config/general';
 import { isAndroid } from '../../../config/system';
+import { TestIdProps } from '../../../interfaces/test-id.props';
 import { formatSize } from '../../../styles/format-size';
 import { generateHitSlop } from '../../../styles/generate-hit-slop';
 import { Icon, IconProps } from '../icon';
 import { TouchableIconStyles } from './touchable-icon.styles';
 
-interface Props extends IconProps {
```
Comment (unixvb) [inline src/components/icon/touchable-icon/touchable-icon.tsx:12]: testId not used

File: src/components/sorter/sorter.tsx [hunk 4]
```diff
@@ -10,7 +11,7 @@ import { Icon } from '../icon/icon';
 import { IconNameEnum } from '../icon/icon-name.enum';
 import { useSorterStyles } from './sorter.styles';
 
-interface Props<T extends string> {
```
Comment (unixvb) [inline src/components/sorter/sorter.tsx:13]: not used

File: src/components/styled-numberic-input/styled-numeric-input.props.ts [hunk 5]
```diff
@@ -1,13 +1,12 @@
...
 import { EventFn } from '../../config/general';
+import { TestIdProps } from '../../interfaces/test-id.props';
 import { StyledTextInputProps } from '../styled-text-input/styled-text-input.props';
 
 export interface StyledNumericInputProps
-  extends Pick<
-    StyledTextInputProps,
-    'editable' | 'placeholder' | 'isError' | 'isShowCleanButton' | 'onBlur' | 'onFocus'
-  > {
+  extends TestIdProps,
```
Comment (unixvb) [inline src/components/styled-numberic-input/styled-numeric-input.props.ts:8]: not used

File: src/components/styled-radio-group/index.tsx [hunk 6]
```diff
@@ -3,14 +3,15 @@ import { View, ViewStyle } from 'react-native';
...
 
+import { TestIdProps } from '../../interfaces/test-id.props';
 import { RadioItemInterface, RadioGroup } from './radio-group';
 import { useStyledRadioButtonsGroupStyles } from './styles';
 
 export interface StyledRadioGroupProps<T extends string> {
   items: RadioItemInterface<T>[];
 }
 
-interface Props<T extends string> extends StyledRadioGroupProps<T> {
```
Comment (unixvb) [inline src/components/styled-radio-group/index.tsx:13]: not used

File: src/components/white-container/white-container-action/white-container-action.tsx [hunk 7]
```diff
@@ -1,9 +1,10 @@
 import React, { FC } from 'react';
 import { TouchableOpacity, TouchableOpacityProps } from 'react-native';
 
+import { TestIdProps } from '../../../interfaces/test-id.props';
 import { WhiteContainerActionStyles } from './white-container-action.styles';
 
-type Props = Pick<TouchableOpacityProps, 'disabled' | 'onPress'>;
```
Comment (unixvb) [inline src/components/white-container/white-container-action/white-container-action.tsx:6]: not used

File: src/components/wallet-address/wallet-address.tsx [hunk 8]
```diff
@@ -15,7 +16,7 @@ import { TouchableIcon } from '../icon/touchable-icon/touchable-icon';
 import { PublicKeyHashText } from '../public-key-hash-text/public-key-hash-text';
 import { useWalletAddressStyles } from './wallet-address.styles';
 
-interface Props {
```
Comment (unixvb) [inline src/components/wallet-address/wallet-address.tsx:18]: not used

File: src/modals/reveal-seed-phrase-modal/reveal-seed-phrase-form-content/reveal-seed-phrase-view/reveal-seed-phrase-view.tsx [hunk 9]
```diff
@@ -3,9 +3,10 @@ import React, { FC, useEffect, useState } from 'react';
 import { OVERLAY_SHOW_TIMEOUT } from '../../../../components/mnemonic/mnemonic.config';
 import { RevealSecretView } from '../../../../components/mnemonic/reveal-secret-view/reveal-secret-view';
 import { useActiveTimer } from '../../../../hooks/use-active-timer.hook';
+import { TestIdProps } from '../../../../interfaces/test-id.props';
 import { useShelter } from '../../../../shelter/use-shelter.hook';
 
-interface Props {
+interface Props extends TestIdProps {
```
Comment (unixvb) [inline src/modals/reveal-seed-phrase-modal/reveal-seed-phrase-form-content/reveal-seed-phrase-view/reveal-seed-phrase-view.tsx:9]: ._.

File: src/modals/select-baker-modal/select-baker-item/select-baker-item.tsx [hunk 10]
```diff
@@ -17,10 +17,11 @@ import { tzktUrl } from 'src/utils/linking.util';
 import { formatToPercentStr } from 'src/utils/number-format.utils';
 import { kFormatter } from 'src/utils/number.util';
 
+import { TestIdProps } from '../../../interfaces/test-id.props';
 import { RECOMMENDED_BAKER_ADDRESS } from '../select-baker-modal';
 import { useSelectBakerItemStyles } from './select-baker-item.styles';
 
-interface Props {
+interface Props extends TestIdProps {
```
Comment (unixvb) [inline src/modals/select-baker-modal/select-baker-item/select-baker-item.tsx:24]: ._.

File: src/modals/select-baker-modal/select-baker-modal.tsx [hunk 11]
```diff
@@ -152,7 +154,11 @@ export const SelectBakerModal: FC = () => {
           />
         </View>
         <View style={styles.searchContainer}>
-          <SearchInput placeholder="Search baker" onChangeText={debouncedSetSearchValue} />
+          <SearchInput
+            placeholder="Search baker"
+            onChangeText={debouncedSetSearchValue}
+            {...setTestID(SelectBakerModalSelectors.searchBakerInput)}
```
Comment (unixvb) [inline src/modals/select-baker-modal/select-baker-modal.tsx:160]: look like this's not correct

File: src/screens/collectibles-home/promotion-carousel/promotion-carousel.selectors.ts [hunk 12]
```diff
@@ -1,6 +1,6 @@
 export enum PromotionCarouselSelectors {
-  QuipuSwap = 'PromotionCarousel/QuipuSwap',
-  Yupana = 'PromotionCarousel/Yupana',
-  SupportUkraine = 'PromotionCarousel/SupportUkraine',
-  MadFishBlog = 'PromotionCarousel/MadFishBlog'
+  quipuSwap = 'Promotion Carousel/QuipuSwap',
+  yupana = 'Promotion Carousel/Yupana',
+  supportUkraine = 'Promotion Carousel/Support Ukraine',
+  madFishBlog = 'Promotion Carousel/MadFish Blog'
```
Comment (unixvb) [inline src/screens/collectibles-home/promotion-carousel/promotion-carousel.selectors.ts:5]: there is not element names

File: src/screens/contacts/contacts.selectors.ts [hunk 13]
```diff
@@ -0,0 +1,4 @@
+export enum ContactsSelectors {
+  addContactButton = 'Settings Contacts/Add Contact "+" Button',
+  contact = 'Settings Contacts/Contact'
```
Comment (unixvb) [inline src/screens/contacts/contacts.selectors.ts:3]: "contact" what?

File: src/screens/d-apps/d-apps.selectors.ts [hunk 14]
```diff
@@ -0,0 +1,5 @@
+export enum DAppsSelectors {
+  integratedDApp = 'DApps/Integrated DApp',
+  othersDApps = 'DApps/Other DApps',
```
Comment (unixvb) [inline src/screens/d-apps/d-apps.selectors.ts:3]: element names are absent

File: src/screens/d-apps/d-apps.tsx [hunk 15]
```diff
@@ -59,7 +61,11 @@ export const DApps = () => {
   return (
     <>
       <InsetSubstitute type="top" />
-      <SearchInput placeholder="Search Dapp" onChangeText={setSearchQuery} />
+      <SearchInput
+        placeholder="Search Dapp"
+        onChangeText={setSearchQuery}
+        {...setTestID(DAppsSelectors.searchDAppsInput)}
```
Comment (unixvb) [inline src/screens/d-apps/d-apps.tsx:67]: the same, you need to pass `testId` prop

File: src/screens/manage-accounts/manage-hd-accounts/manage-hd-accounts.tsx [hunk 16]
```diff
@@ -46,7 +48,11 @@ export const ManageHdAccounts = () => {
 
   return (
     <>
-      <SearchInput placeholder="Search accounts" onChangeText={debouncedSetSearch} />
+      <SearchInput
+        placeholder="Search accounts"
+        onChangeText={debouncedSetSearch}
+        {...setTestID(ManageHdAccountsSelectors.searchAccountsInput)}
```
Comment (unixvb) [inline src/screens/manage-accounts/manage-hd-accounts/manage-hd-accounts.tsx:54]: the same

File: src/screens/manage-accounts/manage-imported-accounts/manage-imported-accounts.tsx [hunk 17]
```diff
@@ -23,7 +25,11 @@ export const ManageImportedAccounts = () => {
 
   return (
     <>
-      <SearchInput placeholder="Search accounts" onChangeText={debouncedSetSearch} />
+      <SearchInput
+        placeholder="Search accounts"
+        onChangeText={debouncedSetSearch}
+        {...setTestID(ManageImportedAccountsSelectors.searchAccountsInput)}
```
Comment (unixvb) [inline src/screens/manage-accounts/manage-imported-accounts/manage-imported-accounts.tsx:31]: and here

File: src/screens/notifications-settings/notifications-settings.selectors.ts [hunk 18]
```diff
@@ -1,3 +1,3 @@
 export enum NotificationsSettingsSelectors {
-  NewsSwitch = 'News Switch on Notifications page is Settings'
+  newsSwitch = 'Settings Notifications/News Toggle'
```
Comment (unixvb) [inline src/screens/notifications-settings/notifications-settings.selectors.ts:2]: different names: "newsSwitch" and "News Toggle"

File: src/screens/secure-settings/secure-settings.selectors.ts [hunk 19]
```diff
@@ -1,5 +1,5 @@
 export enum SecureSettingsSelectors {
-  AnalyticsSwitch = 'Analytics Switch on Secure page is Settings',
-  BiometricsSwitch = 'Biometrics Switch on Secure page is Settings',
-  HideModeOnLaunchSwitch = 'Hide Mode on Launch Switch on Secure page is Settings'
+  analyticsSwitch = 'Settings Secure/Analytics Toggle',
+  biometricsSwitch = 'Settings Secure/Biometrics Toggle',
```
Comment (unixvb) [inline src/screens/secure-settings/secure-settings.selectors.ts:3]: the same

File: src/screens/settings/settings-header/social-button/social-button.tsx [hunk 20]
```diff
@@ -3,11 +3,12 @@ import React, { FC } from 'react';
 
 import { Icon } from '../../../../components/icon/icon';
 import { IconNameEnum } from '../../../../components/icon/icon-name.enum';
+import { TestIdProps } from '../../../../interfaces/test-id.props';
 import { formatSize } from '../../../../styles/format-size';
 import { openUrl } from '../../../../utils/linking.util';
 import { useSocialButtonStyles } from './social-button.styles';
 
-interface Props {
```
Comment (unixvb) [inline src/screens/settings/settings-header/social-button/social-button.tsx:10]: not used

File: src/components/bottom-sheet/bottom-sheet-action-button/bottom-sheet-action-button.tsx (hunk 1 above)
Comment (NovakVlad2424) [reply src/components/bottom-sheet/bottom-sheet-action-button/bottom-sheet-action-button.tsx:8]: deleted

File: src/modals/select-baker-modal/select-baker-modal.tsx (hunk 11 above)
Comment (NovakVlad2424) [reply src/modals/select-baker-modal/select-baker-modal.tsx:160]: fixed

File: src/screens/collectibles-home/promotion-carousel/promotion-carousel.selectors.ts (hunk 12 above)
Comment (NovakVlad2424) [reply src/screens/collectibles-home/promotion-carousel/promotion-carousel.selectors.ts:5]: 'Banner' element is added

File: src/screens/contacts/contacts.selectors.ts (hunk 13 above)
Comment (NovakVlad2424) [reply src/screens/contacts/contacts.selectors.ts:3]: contact => contactItem = 'Settings Contacts/Contact Item'

File: src/screens/d-apps/d-apps.selectors.ts (hunk 14 above)
Comment (NovakVlad2424) [reply src/screens/d-apps/d-apps.selectors.ts:3]: intergratedDApp is button, othersDApps is item

File: src/screens/d-apps/d-apps.tsx (hunk 15 above)
Comment (NovakVlad2424) [reply src/screens/d-apps/d-apps.tsx:67]: fixed

File: src/screens/notifications-settings/notifications-settings.selectors.ts (hunk 18 above)
Comment (NovakVlad2424) [reply src/screens/notifications-settings/notifications-settings.selectors.ts:2]: Toggle

File: src/screens/wallet/collectibles-home-swipe-button/collectibles-home-swipe-button.selectors.ts [hunk 21]
```diff
@@ -0,0 +1,3 @@
+export enum CollectiblesHomeSwipeButtonSelectors {
+  collectiblesButton = 'Wallet/<<<COLLECTIBLES<<< Button'
```
[the remark this answers is not in the document]
Comment (NovakVlad2424) [reply src/screens/wallet/collectibles-home-swipe-button/collectibles-home-swipe-button.selectors.ts:2]: <<<COLLECTIBLES<<< => COLLECTIBLES

File: src/components/styled-numberic-input/styled-numeric-input.tsx [hunk 22]
```diff
@@ -39,6 +41,7 @@ export const StyledNumericInput: FC<StyledNumericInputProps> = ({
       onBlur={handleBlur}
       onFocus={handleFocus}
       onChangeText={handleChange}
+      {...setTestID(testID)}
```
Comment (unixvb) [inline src/components/styled-numberic-input/styled-numeric-input.tsx:44]: testID={testID}

File: src/components/styled-radio-group/index.tsx [hunk 23]
```diff
@@ -29,6 +31,7 @@ export const StyledRadioGroup = <T extends string>({ value, items, onChange, lab
         color={colors.orange}
         itemContainerStyle={styles.itemContainer}
         itemLabelStyle={[styles.label, labelStyle]}
+        {...setTestID(testID)}
```
Comment (unixvb) [inline src/components/styled-radio-group/index.tsx:34]: the same

File: src/form/form-asset-amount-input/form-asset-amount-input.tsx [hunk 24]
```diff
@@ -58,6 +62,7 @@ export const FormAssetAmountInput: FC<Props> = ({
         setSearchValue={setSearchValue}
         onBlur={handleBlur}
         onValueChange={handleValueChange}
+        {...setTestID(testID)}
```
Comment (unixvb) [inline src/form/form-asset-amount-input/form-asset-amount-input.tsx:65]: the same

File: src/modals/reveal-seed-phrase-modal/reveal-seed-phrase-form-content/reveal-seed-phrase-view/reveal-seed-phrase-view.tsx [hunk 25]
```diff
@@ -31,5 +33,7 @@ export const RevealSeedPhraseView: FC<Props> = ({ publicKeyHash }) => {
     });
   };
 
-  return <RevealSecretView value={seedPhrase} onProtectedOverlayPress={handleProtectedOverlayPress} />;
+  return (
+    <RevealSecretView value={seedPhrase} onProtectedOverlayPress={handleProtectedOverlayPress} {...setTestID(testID)} />
```
Comment (unixvb) [inline src/modals/reveal-seed-phrase-modal/reveal-seed-phrase-form-content/reveal-seed-phrase-view/reveal-seed-phrase-view.tsx:37]: the same

File: src/modals/select-baker-modal/baker-list-item/baker-list-item.tsx [hunk 26]
```diff
@@ -5,17 +5,19 @@ import { Divider } from 'src/components/divider/divider';
...
   item: BakerInterface;
   selected: boolean;
   onPress: EventFn<BakerInterface>;
 };
 
-export const BakerListItem = memo<BakerListItemProps>(({ item, selected, onPress }) => (
+export const BakerListItem = memo<BakerListItemProps>(({ item, selected, onPress, testID }) => (
   <>
-    <SelectBakerItem baker={item} selected={selected} onPress={() => onPress(item)} />
+    <SelectBakerItem baker={item} selected={selected} onPress={() => onPress(item)} {...setTestID(testID)} />
```
Comment (unixvb) [inline src/modals/select-baker-modal/baker-list-item/baker-list-item.tsx:20]: the same

**Review by unixvb — APPROVED** [review]

**Pull request merged**